Dockerfile

```diff
@@ -1,81 +1,90 @@
 # syntax=docker/dockerfile:1.4
-# This needs to be bullseye-slim because the Ruby image is built on bullseye-slim
-ARG NODE_VERSION="16.17.1-bullseye-slim"
 
-FROM ghcr.io/moritzheiber/ruby-jemalloc:3.0.4-slim as ruby
-FROM node:${NODE_VERSION} as build
+FROM ruby:3.1.2-slim-bullseye as build
```

Comment:
This won't work. The official ruby images don't have jemalloc, which is why the current version pulls a custom image.

Reply:
@ineffyble While they do not have it, you can install and then enable it:

https://github.com/mastodon/mastodon/pull/21061/commits/204ecbe84e0e96c6e651042eeb7642bb8d6e4f63#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R68

https://github.com/mastodon/mastodon/pull/21061/commits/204ecbe84e0e96c6e651042eeb7642bb8d6e4f63#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R87